packages/calypso-e2e/src/lib/pages/media-page.ts

```diff
@@ -19,6 +24,8 @@ const selectors = {
 	items: '.media-library__list-item',
 	placeholder: '.is-placeholder',
 	editButton: 'button[data-e2e-button="edit"]',
+	addNewButton: '.media-library__upload-button-input',
+	uploadRejectionNotice: 'text=could not be uploaded because the file type is not supported',
```

Comment:
We can be a bit less specific/brittle here
```suggestion
	uploadRejectionNotice: 'text=/could not be uploaded/i',
```